src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/SafeBuffer.cs

```diff
@@ -153,6 +153,8 @@ public void AcquirePointer(ref byte* pointer)
             if (_numBytes == Uninitialized)
                 throw NotInitialized();
 
+            pointer = null;
```

Comment:
Could you add pragma warning around this with reference to the issue?

Reply:
I'm not a big fan of adding suppression for things we don't have enabled.  We'd have an absolute ton of such suppression across SPC and the rest of the repo if we did that.